console/commands/setup.sh

```diff
@@ -174,11 +174,21 @@ if [ "${PHP_VERSION}" != "${DEFAULT_PHP_VERSION}" ]; then
     printf "${COLOR_RESET}\n"
 fi
 
+read -p "Docker compose project name: [${MAGENTO_DIR}] " ANSWER_PROJECT_NAME
+PROJECT_NAME=${ANSWER_PROJECT_NAME-}
```

Comment:
I think it should be shown that it's  optional

Reply:
Any idea on how to show that its optional? Any standarised sentece or something? (some markup)

Reply:
Maybe “Docker compose project name (optional):”?

Reply:
Ok nice! I've updated it!